pkg/reconciler/apis/apiexport/apiexport_controller.go

```diff
@@ -86,6 +89,9 @@ func NewController(
 			_, err := kubeClusterClient.CoreV1().Secrets(secret.Namespace).Create(logicalcluster.WithCluster(ctx, clusterName), secret, metav1.CreateOptions{})
 			return err
 		},
+		getAPIBindingsForAPIExport: func(clusterPathAndName string) ([]interface{}, error) {
+			return apiBindingInformer.Informer().GetIndexer().ByIndex(indexers.APIBindingsByAPIExport, clusterPathAndName)
```

Comment:
By asking for one input here you're not closing over the implementation detail of how the indexer is called, so I'd suggest if possible asking for `func(export *apisv1alpha1.APIExport)` or `func(cluster logicalcluster.Name, name string)`

Reply:
Good call, will fix.